README.md

```diff
@@ -38,19 +38,22 @@ app.listen(3000)
 
 ## Plugin Options
 
-* `algorithm`: all hashing algorithm that Node.js support, and
-  `'fnv1a'`. Default: `'fnv1a'`.
+* `algorithm`: all hashing algorithm that Node.js support, with the addition of
+  `'fnv1a'` and `'xxhash'`. Default: `'fnv1a'`.
 
 ## Acknowledgements
 
 The fnv1a logic was forked from https://github.com/sindresorhus/fnv1a
 and adapted to support buffers.
 
+The xxHash32 algorithm is implemented in pure JavaScript and it's coming through a module written by Jason3S: https://github.com/Jason3S/xxhash.
```

Comment:
https://github.com/Jason3S/xxhash => `[https://github.com/Jason3S/xxhash](https://github.com/Jason3S/xxhash)`

Reply:
Good point, I'll fix also the one above :)